src/util/LazyJsonParser.h

```diff
@@ -20,15 +20,19 @@ namespace ad_utility {
  */
 class LazyJsonParser {
  public:
+  // Generator detail, the first 100 input characters for better error context.
+  struct Details {
+    std::string first100_;
```

Comment:
I think we also want to have the last 100 characters in addition,
it might also happen, that everything starts alright, but in the middle something goes wrong, we also need context for that.